### PR TITLE
Update rules

### PR DESCRIPTION
**Pull request by na-ji**
# Documentation
I've updated these rules:
- Naming convention now allows components name in function parameters.
- Properties can now be kebab case, it's specially useful for HTTP headers or HTML element attributes.

Also, I've added those rules to test files:
- It's not recommended to comment out a test
- It's forbidden to focus a test with `.only`
- It's forbidden to have two tests with the same title (the test runner only run the first one it finds)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -35,7 +35,8 @@ module.exports = {
         selector: 'parameter',
         // allow parameters with leading underscore
         leadingUnderscore: 'allow',
-        format: ['camelCase'],
+        // PascalCase for components, camelCase for everything else
+        format: ['camelCase', 'PascalCase'],
```
Comment (amarlankri) [inline index.js:39]: I don't see a case where we had to pass a component as parameter of a function but I guess you did. Could you show an example @na-ji?

File: index.js [hunk 2]
```diff
@@ -79,6 +79,24 @@ module.exports = {
...
+        '**/*.test.tsx',
+        '**/*.spec.tsx',
+        '**/*.cy.ts',
+        '**/*.cy.tsx',
+      ],
+      plugins: ['jest'],
+      rules: {
+        'jest/no-commented-out-tests': 'warn',
+        'jest/no-focused-tests': 'error',
+        'jest/no-identical-title': 'error',
```
Comment (amarlankri) [inline index.js:96]: 🎉

**Review by amarlankri — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (na-ji) [reply index.js:39]: There are two cases:
- Storybook decorators

https://github.com/algoan/frontend/blob/bb06e9e5eb5af4215009b24068c4f38718b511e3/apps/bank-data/.storybook/preview.tsx#L11-L18

- NextJS App component

https://github.com/algoan/frontend/blob/609075e969cd1d0aa211130a56592ab38080b7a5/apps/bank-data/src/pages/_app.tsx#L37

I don't know if those two cases are enough to tweak the rules, though. What do you think?

**Review by slatchma — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (amarlankri) [reply index.js:39]: I don't know 🤔 But let's see if allowing this rule creates more issues than it solves.

**Review by KenzaYahia — APPROVED** [review]

**Review by g-ongenae — APPROVED** [review]

**Review by kajaksic — APPROVED** [review]

**Review by charlies96 — APPROVED** [review]

**Pull request merged**